src/io/hcg/rgb2hcg.js

```diff
@@ -1,12 +1,12 @@
 const {unpack} = require('../../utils');
 
 const rgb2hcg = (...args) => {
-    const [r,g,b] = unpack(args, 'rgb');
+    const [r,g,b] = unpack(args, 'rgb').map(x => x / 255);
     const min = Math.min(r, g, b);
     const max = Math.max(r, g, b);
     const delta = max - min;
-    const c = delta * 100 / 255;
-    const _g = min / (255 - delta) * 100;
+    // const c = delta;
```

Comment:
I think you can just elliminate these comments. Git history exists.

P.S.: who on Earth decided that calling a color space HCG is a good idea? The medical part of my brain is screaming.

Reply:
Four months later, I thought for sure this repository was dead. Sure I'll remove the comments.

Reply:
I am not a maintainer or anything, so…